src/sys/kernel/mod.rs

```diff
@@ -119,6 +119,27 @@ impl cmp::PartialOrd for Version {
     }
 }
 
+/// Returns the maximum process ID number.
+///
+/// This is taken from `/proc/sys/kernel/pid_max`.
+///
+/// # Example
+///
+/// ```
+/// let pid_max = procfs::sys::kernel::pid_max().unwrap();
+///
+/// let pid = 42; // e.g. from user input, CLI args, etc.
+///
+/// if pid > pid_max {
+///     println!("good process ID: {}", pid);
+/// } else {
+///     eprintln!("bad process ID: {}", pid)
+/// }
+/// ```
+pub fn pid_max() -> ProcResult<libc::pid_t> {
```

Comment:
Can you change the return type to be an `i32` instead of `libc::pid_t`?   I know that `pid_t` is actually the correct return type, but unfortunately I use `i32` everywhere else (for example [here](https://docs.rs/procfs/0.7.9/procfs/process/struct.Process.html#structfield.pid), [here](https://docs.rs/procfs/0.7.9/procfs/process/struct.Stat.html#structfield.pid), and [here](https://docs.rs/procfs/0.7.9/procfs/process/struct.MountInfo.html#structfield.pid)).   Eventually I might change everything into a `pid_t`, but until then I'd prefer all the pid types to be consistent.

Reply:
Done.